metrics/format_json.py

```diff
@@ -6,11 +6,17 @@
 data = json.load(sys.stdin)
 sep = "module.exports = {\n"
 for font in sorted(data):
-  sys.stdout.write(sep + json.dumps(font))
-  sep = ": {\n  "
-  for glyph in sorted(data[font], key=int):
-      sys.stdout.write(sep + json.dumps(glyph) + ": ")
-      sys.stdout.write(json.dumps(data[font][glyph], sort_keys=True))
-      sep = ",\n  "
-  sep = "\n},\n"
-sys.stdout.write("\n}};\n");
+    sys.stdout.write(sep + json.dumps(font))
+    sep = ": {\n  "
+    for glyph in sorted(data[font], key=int):
```

Comment:
I tried it without and it uses string comparison to order the entries so "100" becomes before "2".